src/main/kotlin/no/nav/familie/ba/sak/integrasjoner/økonomi/utbetalingsoppdrag/UtbetalingsoppdragGenerator.kt

```diff
@@ -43,12 +45,34 @@ class UtbetalingsoppdragGenerator(
                 erSimulering,
             )
 
+        val skalBrukeNyKlassekodeForUtvidetBarnetrygd =
+            unleashNextMedContextService.isEnabled(
+                toggleId = FeatureToggleConfig.SKAL_BRUKE_NY_KLASSEKODE_FOR_UTVIDET_BARNETRYGD,
+                behandlingId = vedtak.behandling.id,
+            )
+
+        val forrigeAndeler =
+            if (forrigeTilkjentYtelse == null) {
+                emptyList()
+            } else if (vedtak.behandling.opprettetÅrsak != BehandlingÅrsak.NY_UTVIDET_KLASSEKODE) {
+                forrigeTilkjentYtelse.tilAndelData(skalBrukeNyKlassekodeForUtvidetBarnetrygd) ?: emptyList()
```

Comment:
```suggestion
                forrigeTilkjentYtelse.tilAndelData(skalBrukeNyKlassekodeForUtvidetBarnetrygd)
```
forrigeTilkjentYtelse kan ikke være null etter første if sjekk, og tilAndelData kan heller ikke returnere null.
Elvis operatøren vil derfor aldri slå inn.